apps/static/dance/ai/model/cached_background_effects_map.json

```diff
@@ -156,5 +156,36 @@
     "starburst",
     "stars",
     "music_wave"
+  ],
+  "modelDescriptiveNames": [
```

Comment:
Is there a reason we're including these now?

Reply:
For testing purposes. I can remove once finalized.

Reply:
Is this not finalized?

Reply:
We may be using a different model. I'll Slack you about details.

Reply:
Not sure if you're still using these, guess doesn't hurt to keep them?

Reply:
We are using these when testing with the .ipynb [testing file](https://drive.google.com/file/d/19n5ELeVS9to8Vvw-q2AMUxeydWWWXtGz/view). Once we finalize model and maps, I'll remove. Thanks!